### PR TITLE
Updated images and added FB button

### PR DESCRIPTION
**Pull request by mira0993**
- Updated all the page images
- Added footer
- Added page like button

### CODE REVIEW

File: cidemipyme-gatsby/src/components/sections/FooterSection.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Container,
+  Col,
+  Row
+} from 'reactstrap';
+
+class FooterSection extends React.Component {
+  constructor(props) {
```
Comment (cdvr1993) [inline cidemipyme-gatsby/src/components/sections/FooterSection.jsx]: why are you declaring the constructor with the default implementation?

File: cidemipyme-gatsby/src/html.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 import React from "react"
+import favicon from 'images/logo_favicon_32.png';
+import preview from 'images/page_preview.png';
+
 
 let stylesStr
```
Comment (cdvr1993) [inline cidemipyme-gatsby/src/html.js]: missing semicolon

File: cidemipyme-gatsby/src/html.js [hunk 3]
```diff
@@ -29,18 +32,40 @@ module.exports = class HTML extends React.Component {
...
+          <meta property="og:url" content="http://cidemipyme1.s3-website-us-west-2.amazonaws.com/" />
+          <meta property="og:image" content={preview} />
+          <link rel="icon" href={favicon}/>
           {this.props.headComponents}
           {css}
         </head>
         <body {...this.props.bodyAttributes}>
+          <div id="fb-root"></div>
+          <script
+            dangerouslySetInnerHTML={{
```
Comment (cdvr1993) [inline cidemipyme-gatsby/src/html.js]: maybe a comment here, just to know what this code is doing

File: cidemipyme-gatsby/src/html.js (hunk 2 above)
Comment (mira0993) [reply cidemipyme-gatsby/src/html.js]: autogenerated code

**Pull request merged**